webcompat/static/js/lib/bugform-prefill.js

```diff
@@ -0,0 +1,57 @@
+/* exported prefillForm */
+function prefillForm(message, isExperiment) {
+  if (!message) return;
+
+  var config = {
+    url: {
+      element: $("#url")
+    },
+    details: {
+      element: $("#details"),
+      stringify: true
+    },
+    src: {
+      element: $("#reported_with")
+    },
+    extra_labels: {
+      element: $("#extra_labels"),
+      stringify: true,
+      label: true
+    }
+  };
+
+  var prepareValue = function(field, value) {
+    if (field.label && isExperiment) {
+      value.push("form-v2-experiment");
+    }
+
+    if (field.stringify) {
+      return JSON.stringify(value);
+    }
+
+    return value;
+  };
+
+  var setValues = function(data) {
+    Object.keys(config).forEach(function(key) {
+      if (data.hasOwnProperty(key)) {
+        var field = config[key];
+        field.element.val(prepareValue(field, data[key]));
+      }
+    });
+  };
+
+  var setAnalytics = function(campaign, source) {
+    if (!window.setAnalyticsData) return;
```

Comment:
This is to account for the inline script that is inserted based on "utm_campaign" and "utm_source" existence in request on prefill stage https://github.com/webcompat/webcompat.com/blob/master/webcompat/views.py#L273